src/inset_state/flatten_density.cpp

```diff
@@ -453,24 +447,26 @@ void InsetState::flatten_density_with_node_vertices()
       //                        y + 0.5*delta_t*v_y(x,y,t),
       //                        t + 0.5*delta_t)
       // and similarly for y.
-      calculate_velocity(
-        t + 0.5 * delta_t,
-        grid_fluxx_init,
-        grid_fluxy_init,
-        rho_ft_,
-        rho_init_,
-        &grid_vx,
-        &grid_vy,
-        lx_,
-        ly_);
+      std::function<double(unsigned int, unsigned int, char)> get_velocity =
```

Comment:
The semantic difference between `cal_velocity` and `get_velocity` is not immediately obvious. Would `cal_velocity_at_half_time` be better than `get_velocity()`?

Reply:
Turns out I missed renaming `get_velocity` to `cal_velocity`. Both serve the same function. So, I feel it is best to let them have the same name.